js/compileShell.py

```diff
@@ -578,6 +578,7 @@ def fixateVer(pcfile):
         """Returns the current version number (47.0a2)."""
         with open(pcfile, 'rb') as f:
             for line in f:
+                line = line.decode("utf-8", "replace")
```

Comment:
You could also accomplish this using the `io` module:

    with io.open(pcfile, encoding="utf-8", errors="replace"):